src/components/StudentGrades/StudentGrades.tsx

```diff
@@ -28,18 +28,18 @@ export const StudentGrades = (props: GradesValuesProps) => {
       className="student-grades"
       sx={{
         fontSize: '14px',
-        fontFamily: 'Catamaran Regular',
```

Comment:
There is Catamaran Regular font on the mock-up. Is Catamaran the same? It can be :)